plugins/outputs/influxdb_v2/influxdb_v2_test.go

```diff
@@ -1360,12 +1362,38 @@ func BenchmarkWrite100k(b *testing.B) {
 }
 
 func BenchmarkWriteConcurrent100k_4(b *testing.B) {
-	batchsize := 100000
+	batchsize := 100001
 
+	var received atomic.Int32
 	// Setup a test server
 	ts := httptest.NewServer(
-		http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
-			w.WriteHeader(http.StatusNoContent)
+		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			switch r.URL.Path {
+			case "/api/v2/write":
+				reader, err := gzip.NewReader(r.Body)
+				if err != nil {
+					w.WriteHeader(http.StatusInternalServerError)
+					b.Error(err)
+					return
+				}
+				defer reader.Close()
+				body, err := io.ReadAll(reader)
+				if err != nil {
+					w.WriteHeader(http.StatusInternalServerError)
+					b.Error(err)
+					return
+				}
+				lineCount := bytes.Count(body, []byte("\n"))
+				if len(body) > 0 && body[len(body)-1] != '\n' {
+					lineCount++
+				}
+				received.Add(int32(lineCount))
+				w.WriteHeader(http.StatusNoContent)
+				return
+			default:
+				w.WriteHeader(http.StatusNotFound)
+				return
+			}
 		}),
```

Comment:
Please create a new test for this instead of changing the **benchmark** as this will distort the benchmark results.